### PR TITLE
mce-install: Adjust 'OVERRIDE_HO_IMAGE' logic and display HO info

### PR DESCRIPTION
**Pull request by LiangquanLi930**

### CODE REVIEW

**Comment by LiangquanLi930** [comment]
/pj-rehearse periodic-ci-openshift-hypershift-release-4.18-periodics-mce-e2e-agent-connected-ovn-ipv4-metal3-compact-conformance

File: ci-operator/step-registry/hypershift/mce/install/hypershift-mce-install-commands.sh [hunk 1]
```diff
@@ -233,10 +233,11 @@ metadata:
 data:
   hypershift-operator: ${OVERRIDE_HO_IMAGE}
 EOF
-  while ! [ "$(oc get deployment operator -n hypershift -o jsonpath='{.status.conditions[?(@.type=="Progressing")].reason}')" == NewReplicaSetAvailable ]; do
+  while ! [ "$(oc get deployment operator -n hypershift -o jsonpath='{.spec.template.spec.containers[*].image}')" == "$OVERRIDE_HO_IMAGE" ]; do
```
Comment (heliubj18) [inline ci-operator/step-registry/hypershift/mce/install/hypershift-mce-install-commands.sh:236]: I prefer [[ ]] instead of [] in the shell script

**Pull request merged**